docs/cellxgene_census_docsite_landing.md

```diff
@@ -1,4 +1,6 @@
-<span style="background-color: #F6DDCC; color; font-size: 18px"> 🚨 A **normalized layer**, plus cell and gene **pre-calculated stats** are now available!  Find out more [here.](articles/2023/20231012-normalized_layer_precalc_stats.md)</span>
+<span style="background-color: #f3bfcb; color; font-size: 18px"> 🚀 New to the Census: we’ve created a **centralized hub of models and embeddings** using Census data. [Check it out](https://cellxgene.cziscience.com/census-models)!
+
+ </span>
```

Comment:
the image below is quite dated wrt statistics for the Dec LTS release. update?

Reply:
good point reminds me I need to update https://chanzuckerberg.github.io/cellxgene-census/cellxgene_census_docsite_data_release_info.html

Reply:
done